documentation/modules/configuring/con-exposing-kafka-connect-api.adoc

```diff
@@ -0,0 +1,66 @@
+// This assembly is included in the following assemblies:
+//
+//assembly-config-kafka-connect.adoc
+
+[id='con-exposing-kafka-connect-api-{context}']
+
+= Exposing the Kafka Connect API
+
+[role="_abstract"]
+Use the Kafka Connect REST API as an alternative to using `KafkaConnector` resources to manage connectors.
```

Comment:
Worth a note:

> NOTE: The `strimzi.io/use-connector-resources` annotation enables KafkaConnectors. 
> If you applied the annotation to your `KafkaConnect` resource configuration, you need to remove it to use the Kafka Connect API. Otherwise, manual changes made directly using the Kafka Connect REST API are reverted by the Cluster Operator.